start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -75,20 +57,84 @@ function EvaluationTableBody(options={}){
     `;
 }
 
-function EvaluationsTable(options={}){
+windowObjectWrap.DetailsEvaluation = function(options={}){
+    
+    return `
+        <div class='textarea-details-wrapper'>
+            <div class='details-list details-textarea'><p>${options.textarea1}</p></div>
+            <div class='details-list details-textarea'><p>${options.textarea2}</p></div>
+            <div class='details-list details-textarea'><p>${options.textarea3}</p></div>
+        </div>
+        <div class='oop-wrapper'>
+            <div class='details-list details-oop'>${options.classes}</div>
+            <div class='details-list details-oop'>${options.exceptionHandling}</div>
+            <div class='details-list details-oop'>${options.versionControl}</div>
+            <div class='details-list details-oop'>${options.accessModifiers}</div>
+            <div class='details-list details-oop'>${options.designPatterns}</div>
+            <div class='details-list details-oop'>${options.issueTracking}</div>
+            <div class='details-list details-oop'>${options.polymorphism}</div>
+            <div class='details-list details-oop'>${options.regex}</div>
+        </div>
+        <div class='http-wrapper'>
+            <div class='details-list details-http'>${options.protocol}</div>
+            <div class='details-list details-http'>${options.responseCodes}</div>
+            <div class='details-list details-http'>${options.rest}</div>
+            <div class='details-list details-http'>${options.headers}</div>
+            <div class='details-list details-http'>${options.requestMethods}</div>
+        </div>
+        <div class='html-wrapper'>
+            <div class='details-list details-html'>${options.doctype}</div>
+            <div class='details-list details-html'>${options.tags}</div>
+            <div class='details-list details-html'>${options.basicSeo}</div>
+            <div class='details-list details-html'>${options.syntaxRules}</div>
+            <div class='details-list details-html'>${options.attributes}</div>
+        </div>
+        
+    `;
+}
+
+windowObjectWrap.Details = function(options={}){
+
+    const detailsElements = options.items.map(el =>{
+        return windowObjectWrap.DetailsEvaluation(el);
+    })
+    //const detailsEl = detailsElements.join('');
+    
+    
+    //console.log(detailsElements);
+    detailsElements.forEach((f,i)=>{
```

Comment:
What does this do?

Reply:
I was just testing to see how many details elements I have. Nothing important.